net/travelmate/files/travelmate.sh

```diff
@@ -291,11 +291,19 @@ f_check()
 								trm_connection="${result:-"-"}/${trm_ifquality}"
 								f_jsnup
 								login_command="$(uci_get "travelmate" "${uci_essid}${uci_bssid}" "command")"
+								login_command_args="$(uci_get "travelmate" "${uci_essid}${uci_bssid}" "command_args")"
 								if [ -x "${login_command}" ]
 								then
-									"${login_command}" >/dev/null 2>&1
-									rc=${?}
-									f_log "info" "captive portal login '${login_command:0:40}' for '${cp_domain}' has been executed with rc '${rc}'"
+									if [ "${login_command_args}" ]
```

Comment:
We don't need the if /else clause at all, it makes no harm if "login_command_args" is empty.